module/userOrganisations/helper.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * name : helper.js
+ * author : Aman
+ * created-date : 11-May-2020
+ * Description : User organisations related informations
+ */
+
+/**
+    * UserOrganisationsHelper
+    * @class
+*/
+
+module.exports = class UserOrganisationsHelper {
+
+    /**
+   * Lists of user organisations
+   * @method
+   * @name list
+   * @param {Object} userIds - list of user ids.
+   * @returns {Object} - key userId and value as organisations and root organisations lists 
+   */
+
+    static list(userIds = "") {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+                let result = {
+                    success : false,
+                    data : {}
+                };
+
+                await Promise.all(userIds.map(async function (userId) {
```

Comment:
@aman-tunerlabs - Can we get for all users in query, please.

Reply:
@akash-above  So for non primary key cassandra does not provide query statement or In functionality which will take an array. That is the reason why i do not used as a single query.